keyboards/handwired/eagleii/rules.mk

```diff
@@ -0,0 +1,29 @@
+# MCU name
+MCU = atmega32u4
+
+# Processor frequency.
+F_CPU = 16000000
+
+# Target architecture (see library "Board Types" documentation).
+ARCH = AVR8
+
+# Input clock frequency.
+F_USB = $(F_CPU)
+
+# Interrupt driven control endpoint task(+60)
+OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+# Boot Section Size in *bytes*
+OPT_DEFS += -DBOOTLOADER_SIZE=4096
```

Comment:
This section is no longer needed, and is set by default. 

However, you should add: 
```
BOOTLOADER = atmel-dfu
```